### PR TITLE
[oppo] Fix issue with polling and setting verbose mode

### PR DESCRIPTION
**Pull request by mlobstein**
After trying the binding in my production system, I discovered a few small issues. Mainly if the player is not on when the binding starts up, setting the desired verbose mode will not work nor do the initial device state query commands. This PR corrects these issues and ensures that the necessary commands are issued after the player is turned on.

### CODE REVIEW

File: bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java [hunk 1]
```diff
@@ -38,6 +38,8 @@
     public static final Integer BDP10X_PORT = 48360;
     public static final Integer BDP20X_PORT = 23;
 
+    public static final String FIRMWARE_PROPERTY = "Firmware Version";
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java:41]: Please use the existing constant for this: https://github.com/openhab/openhab-core/blob/5ab248c158164bf68f020192e98d0d4ccb8010fc/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L55

**Review by kaikreuzer — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java]: done

**Review by cpmeister — APPROVED** [review]
LGTM

**Comment by fwolter** [comment]
Cherry-picking to `2.5.x` is not possible, because `PROPERTY_FIRMWARE_VERSION` doesn't exist there. If you want to have this fix in 2.5.x, please file another PR.

**Pull request merged**